src/main/java/org/folio/service/finance/transaction/FinanceUtils.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.service.finance.transaction;
+
+import lombok.experimental.UtilityClass;
+import org.folio.rest.acq.model.finance.Encumbrance;
+import org.folio.rest.acq.model.finance.Transaction;
+import org.javamoney.moneta.Money;
+import org.javamoney.moneta.function.MonetaryFunctions;
+
+import javax.money.CurrencyUnit;
+import javax.money.MonetaryAmount;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.function.Function;
+
+@UtilityClass
+public class FinanceUtils {
+
+  public static double getEncumbranceExpended(Transaction transaction) {
+    return getExpended(transaction.getEncumbrance());
+  }
+
+  public static double getExpended(Encumbrance encumbrance) {
+    return encumbrance == null ? 0 : encumbrance.getAmountExpended() - encumbrance.getAmountCredited();
+  }
```

Comment:
Need to verify one more time, I believe we shouldn't get expended as amountExpended - amountCredited.

Reply:
Multiple pr to understand context more.
https://github.com/folio-org/mod-finance-storage/pull/417
https://github.com/folio-org/mod-finance-storage/pull/414
https://github.com/folio-org/mod-finance/pull/252

Reply:
I believe this is correct. Were you able to verify?

Reply:
https://github.com/folio-org/mod-finance-storage/pull/414#discussion_r1638157362.
Look at code review comment from Damien.

Reply:
Again, please check where we use this calculation:
1. We need to add credited amount to `poLineEstimatedPriceAfterRollover` when calculating `BasedOn.EXPENDED` right?
2. When we validate encumbrance for deletion now we need to check if `expended - credited > 0` right?
I don't see Damien's comment is applicable here.

Reply:
I mean method name. getExpended() should return expended itself. I think you need to redesign the code to get individual fields.

Reply:
renamed

Reply:
Answer to this question is in https://folio-org.atlassian.net/browse/MODFISTO-483 comments.
So now in PR it is implemented as expected